packages/react-theme-sass/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@fluentui/react-theme-sass",
+  "version": "9.0.0-beta.3",
+  "description": "SASS variables referencing react-theme design tokens injected to DOM by react-provider.",
+  "private": true,
+  "main": "lib/index.js",
+  "typings": "lib/index.d.ts",
```

Comment:
These should probably be removed (along with the index file) since this package isn't for JS